File: Typescript.js/Card.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.Card = void 0;
var Card = /** @class */ (function () {
    function Card(content, wascorrect) {
        this.content = content;
        this.wascorrect = wascorrect;
    }
    Card.prototype.getcontent = function () {
        return this.content;
    };
    Card.prototype.wasCorrectInLastRound = function () {
        return this.wascorrect;
    };
    Card.prototype.toString = function () {
        return this.content;
    };
    return Card;
}());
exports.Card = Card;

File: Java/Card.java
package Java;

public class Card {

    private String content;
    private boolean wascorrect;

    public Card(String cardText, boolean wascorrect) {
        this.content = cardText;   
        this.wascorrect = wascorrect;
    }

    public boolean wascorrectInLastRound() {
        return wascorrect;
    }

    public String getCardText() {
        return content;
    }

    @Override
    public String toString() {
        return content + "  ("+ wascorrectInLastRound()+ ")" ;
    }
}


